src/Examine.Lucene/Providers/LuceneIndex.cs

```diff
@@ -49,6 +59,18 @@ public class LuceneIndex : BaseIndexProvider, IDisposable, IIndexStats, Referenc
             //initialize the field types
             _fieldValueTypeCollection = new Lazy<FieldValueTypeCollection>(() => CreateFieldValueTypes(_options.IndexValueTypesFactory));
 
+            if (_options.UseTaxonomyIndex)
+            {
+                _taxonomySearcher = new Lazy<LuceneTaxonomySearcher>(CreateTaxonomySearcher);
+
+                _searcher = new Lazy<BaseLuceneSearcher>(() => _taxonomySearcher.Value);
+            }
+            else
+            {
+                _taxonomySearcher = new Lazy<LuceneTaxonomySearcher>(() => throw new NotSupportedException("TaxonomySearcher not supported when not using taxonomy index."));
+                _searcher = new Lazy<BaseLuceneSearcher>(CreateSearcher);
+            }
+
```

Comment:
@nzdev If I remember correctly you added this protected constructor. It was complaining that the `_searcher` wasn't set. So I've added the same code as from the private constructor. Do you believe this is the right course of action?

Reply:
Looks good to me.